Unable to deploy new API versions

Deployment fails [here](https://github.com/PolicyEngine/policyengine-household-api/actions/runs/16635432091/job/47075044729).

## What's happening?
The "Deploy" job on push runs the `make deploy` script. The output logs report a series of 14 build steps that appear to be part of a Docker image deploy, and on step 13, `make install` fails with the following error message:
```
Step #1: ERROR: Package '***-household-api' requires a different Python: 3.10.14 not in '>=3.13'
Step #1: make: *** [Makefile:2: install] Error 1
```

--- a/changelog_entry.yaml
+++ b/changelog_entry.yaml
@@ -1,0 +1,4 @@
+- bump: patch
+  changes:
+    changed:
+    - Update Docker build address to use GitHub Container Registry

--- a/gcp/policyengine_household_api/Dockerfile
+++ b/gcp/policyengine_household_api/Dockerfile
@@ -1,4 +1,4 @@
-FROM policyengine/policyengine-household-api:latest
+FROM ghcr.io/policyengine/policyengine-household-api:latest
 
 ENV GOOGLE_APPLICATION_CREDENTIALS .gac.json
 ENV AUTH0_ADDRESS_NO_DOMAIN .address

